docker/base/Dockerfile

```diff
@@ -26,26 +26,12 @@ ENV LANGUAGE en_US.UTF-8
 
 RUN apt-get update && apt-get dist-upgrade -y && \
 	apt-get install -y sudo curl unzip software-properties-common apt-transport-https git bzip2 wget locales && \
-	add-apt-repository ppa:cwchien/gradle && \
     locale-gen en_US.UTF-8 && \
-
-##################
-#    MiniConda   #
-##################
     echo 'export PATH=/opt/conda/bin:$PATH' > /etc/profile.d/conda.sh && \
     wget --quiet https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh && \
     /bin/bash ~/miniconda.sh -b -p /opt/conda && \
-    rm ~/miniconda.sh
-
-##############
-#  Node Java #
-##############
-RUN conda install -y -c conda-forge nodejs openjdk
```

Comment:
i thought you kept the conda version and removed the apt-get version?

Reply:
i wrongly said
i removed installation for whole ```conda nodejs and openjdk``` but left installation for specific environment in setup.sh https://github.com/twosigma/beakerx/blob/michal/6356/docker/setup.sh